frontend/src/components/penreg/student-search/StudentSearchDisplay.vue

```diff
@@ -310,7 +291,7 @@ export default {
       if(validationRequired === false || (this.$refs.studentSearchForm.validate() && this.searchHasValues())) {
         const studentSearchKeys = Object.keys(this.studentSearchParams).filter(k => (this.studentSearchParams[k] && this.studentSearchParams[k].length !== 0));
         let studentSearchFilters;
-        if (studentSearchKeys && studentSearchKeys.length > 0) {
+        if (studentSearchKeys && studentSearchKeys.length >1) {
```

Comment:
we could do if(studentSearchKeys?.length >1)

Reply:
I will fix those now, thank you Om